x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/agent_logs.tsx

```diff
@@ -336,15 +302,25 @@ export const AgentLogsUI: React.FunctionComponent<AgentLogsProps> = memo(
           </EuiFlexGroup>
         </EuiFlexItem>
         <EuiFlexItem>
-          <EuiPanel paddingSize="none" panelRef={logsPanelRef} grow={false}>
-            <LogStream
-              logView={LOG_VIEW_SETTINGS}
-              height={logPanelHeight}
-              startTimestamp={dateRangeTimestamps.start}
-              endTimestamp={dateRangeTimestamps.end}
-              query={logStreamQuery}
-              columns={LOG_VIEW_COLUMNS}
-            />
+          <EuiPanel paddingSize="none" grow={false}>
+            {logSources.value ? (
+              <LazySavedSearchComponent
+                dependencies={{ embeddable, searchSource, dataViews }}
+                index={logSources.value}
+                timeRange={{
+                  from: state.start,
+                  to: state.end,
+                }}
+                query={logStreamQuery}
+                height={'60vh'}
+                displayOptions={{
+                  solutionNavIdOverride: null,
+                  enableDocumentViewer: true,
+                  enableFilters: false,
+                }}
+                columns={['@timestamp', 'message']}
```

Comment:
can we keep the columns LOG_VIEW_COLUMNS?

```suggestion
                columns={['@timestamp', 'event.dataset', 'component.id', 'message', 'error.message']}
```

Reply:
That was the idea when discussed with @kpollich a while back.

Reply:
@juliaElastic Reverted the column change